### PR TITLE
Spinner: Convert Spinner to web component

### LINKED ISSUE #3642: [WebComp] Convert Spinner to web component

### PR DESCRIPTION
**Pull request by jakobe**
## Which issue does this PR close?

This PR closes #3642

## What is the new behavior?

Added new implementation of `Spinner` as a web component.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/angular/spinner/src/spinner.component.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'kirby-spinner',
+  template: '<ng-content></ng-content>',
+  standalone: true,
+  imports: [CommonModule],
+})
+export class KirbySpinnerComponent {}
```
Comment (Fuzzy3) [inline libs/angular/spinner/src/spinner.component.ts]: Do we need to detach ChangeDetectorRef, even though we don't have any attributes to proxy?

**Review by Fuzzy3 — CHANGES_REQUESTED** [review]
Reaaaally nice

File: libs/core/src/spinner/spinner.element.styles.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  .inner-circle,
+  .outer-circle {
+    width: 100%;
+    height: 100%;
+    border-radius: var(--kirby-border-radius-circle);
+    background-color: var(--kirby-primary);
+    opacity: 0.6;
+    position: absolute;
+    top: 0;
+    left: 0;
```
Comment (RasmusKjeldgaard) [inline libs/core/src/spinner/spinner.element.styles.ts]: I think these are redundant with width and height set to 100%

**Review by RasmusKjeldgaard — COMMENTED** [review]
One smol css cleanup suggestion is all, otherwise it LGTM 👍

**Review by Fuzzy3 — APPROVED** [review]

**Pull request merged**